frontend/packages/kubevirt-plugin/src/components/modals/uninstall-kubevirt-modal/uninstall-kubevirt-message.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import { ExternalLink, FirehoseResult } from '@console/internal/components/utils';
+import { connect } from 'react-redux';
+import { FormGroup, TextInput } from '@patternfly/react-core';
+import { VMIKind, VMKind } from '../../../types/vm';
+import { getBasicID, getLoadedData } from '../../../utils';
+import * as _ from 'lodash';
+
+const CONFIRMATION_TEXT = 'kubevirt-hyperconverged-operator';
+const NO_VMS_FOUND_MESSAGE = 'No virtual machines found on this cluster';
+
+const INCORRECT_CONFIRMATION_TEXT = 'Entered text does not match required text';
+const MISSING_CONFIRMATION_TEXT = 'Please enter confirmation text';
+
+const getNumberOfVMsOnCluster = (
+  vms: FirehoseResult<VMKind[]>,
+  vmis: FirehoseResult<VMIKind[]>,
+): number => {
+  const loadedVMs = getLoadedData(vms);
+  const loadedVMIs = getLoadedData(vmis);
```

Comment:
`getLoadedData` can return `null` if data is not yet loaded, so this can be a little dangerous:
you can add a `array` default value, and ignore the possibility of falsely missing unloaded data, or  wait for data to be loaded before allowing to continue. ( my vote is for option b )

https://github.com/openshift/console/blob/master/frontend/packages/kubevirt-plugin/src/utils/index.ts#L32
https://lodash.com/docs/4.17.15#unionBy

Reply:
Good point. _.unionBy would behave normally if `null` was passed for one of the arrays, but it would probably cause an error if both came back as `null`. I'll look into option b, as suggested.